halloffame.php

```diff
@@ -28,18 +22,43 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" type="text/css" href="halloffame.css.css">
+    <link rel="stylesheet" type="text/css" href="halloffame.css">
+
     <title>halloffame</title>
 </head>
 <body>
-<h1> FOOTBALLERS HALL OF FAME  </h1>
-<div id="container"
+<div class="navTop">
+    <div class="<img src="halloffame.png"></div>
+    <div class="title">
+        <h1> FOOTBALLERS HALL OF FAME </h1>
+    </div>
+</div>
 
+<header>
+<class="logo"> <img src="football.jpg" alt= "hall of fame logo"/>
+</header>
+
+<h3> H.O.F Nominees </h3>
+
+<div id="container"
 <div class="Players">
-    <h3> PLAYERS</h3>
-    <p> <?php echo "ronaldo"; ?></p>
-    <p> <?php echo "van persie"; ?></p>
-    <p> <?php echo "evra"; ?></p>
-    <p> <?php echo "rooney"; ?></p>
+
+    <?php foreach ($fame as $player) {
+        echo "<div class='image'>";
+        echo "<img src='" . $player['image'] . "'/>";
+        echo "</div>";
+        echo "<div class='player'>";
+      echo "<div> {$player['name']} <br> </div>";
+      echo "<div> {$player['age']} <br> </div>";
+      echo "<div> {$player['nationality']} <br> </div>";
+      echo "<div> {$player['goal']} <br> </div>";
+        echo "</div>";
+    }?>
+
```

Comment:
Just saying it because Ash said it to me - could wrap this foreach loop inside a function - save the function in a different file, and then call the function inside these <?php tags?
so literally just wrap all your foreach loop inside: 
function displayPlayerStats($fame) { 
       <foreachstuffhere> 
}
then save the whole thing in a different file. 
Require in the file, e.g. require_once './displayPlayerStatsFunction';
Then in the <?php tags in your html, call the function:
displayPlayerStats($fame);
Let me know if that doesn't make sense